### PR TITLE
Report restructure - Fixes

### PR DESCRIPTION
**Pull request by ifarzana**

### CODE REVIEW

File: lib/routes.php [hunk 1]
```diff
@@ -356,7 +356,6 @@
  * )
  */
 $app->get('/stats', ['\Xibo\Controller\Stats','grid'])->setName('stats.search');
-$app->get('/stats/timeDisconnected', ['\Xibo\Controller\Stats', 'gridTimeDisconnected'])->setName('stats.timeDisconnected.search');
```
Comment (dasgarner) [inline lib/routes.php:359]: This is an API which we put in for a user request

File: lib/Controller/Stats.php [hunk 2]
```diff
@@ -837,430 +836,6 @@ public function export(Request $request, Response $response)
...
-     *      )
-     *  )
-     * )
-     * @param Request $request
-     * @param Response $response
-     * @return \Psr\Http\Message\ResponseInterface|Response
-     * @throws InvalidArgumentException
-     * @throws \Xibo\Support\Exception\GeneralException
-     */
-    public function gridTimeDisconnected(Request $request, Response $response): Response
```
Comment (dasgarner) [inline lib/Controller/Stats.php:1150]: This is an API we put in for a user request (hence the API documentation, etc)

**Comment by ifarzana** [comment]
This Pull Request includes: 

- [x] Summary Report: Saved report conversion from old schema to new schema returned an error `Undefined index for Period Start' - This problem is due to very old reportschedule.json file. The conversion works for latest saved reports.
- [x] Refined Category names 
- [x] Proof of Play report fix 
- [x] Time disconnected report showing 200% for a player having two null displayevents.

File: lib/routes.php (hunk 1 above)
Comment (ifarzana) [reply lib/routes.php:359]: If I put this back, I think that is fine. We are not using this method in any of our reports

File: lib/Controller/Stats.php (hunk 2 above)
Comment (ifarzana) [reply lib/Controller/Stats.php:1150]: If I put this back, I think that is fine.

**Pull request merged**